weave/trace_server/trace_server_interface.py

```diff
@@ -1509,6 +1509,115 @@ class ScorerDeleteV2Res(BaseModel):
     num_deleted: int = Field(..., description="Number of scorer versions deleted")
 
 
+class EvaluationCreateV2Body(BaseModel):
+    name: str = Field(
+        ...,
+        description="The name of this evaluation.  Evaluations with the same name will be versioned together.",
+    )
+    description: Optional[str] = Field(
+        None,
+        description="A description of this evaluation",
+    )
+
+    dataset_ref: str = Field(..., description="Reference to the dataset (weave:// URI)")
+    scorer_refs: Optional[list[str]] = Field(
+        None, description="List of scorer references (weave:// URIs)"
+    )
```

Comment:
Here we could expect an optional 
```python
    op_refs: Optional[dict["evaluate" | "summarize" | "predict_and_score", str]]
```

It could be optional so that when we switch to the new data model, we don't expect these anymore from the SDK.